packages/eslint-plugin-okta/lib/rules/no-exclusive-language.test.js

```diff
@@ -14,6 +14,8 @@ ruleTester.run('no-exclusive-language', rule, {
     '\'banlist\'',
     '\'block list\'',
     '\'blocked\'',
+    '\'masterson\'',
+    '\'lemaster\'',
```

Comment:
these are words that would have previously have triggered the linting rule but should not have.